New Theme

seanmarcia opened the pull request:
### What github issue is this PR for, if any?
Resolves #2XXX

### What changed, and why?


### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪


### Screenshots please :)


### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`![alt text](https://media.giphy.com/media/1nP7ThJFes5pgXKUNf/giphy.gif)`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

seanmarcia committed:
rebase

compwron commented on app/helpers/application_helper.rb:
```diff
@@ -22,6 +22,14 @@ def default_page_header
+  def session_signout_link
+    if user_signed_in?
+      destroy_user_session_path
+      elsif all_casa_admin_signed_in?
```
the linter might dislike this

seanmarcia replied:
Definitely. This was mostly to get the sign out working.
There will need to be some refactor issues for this, the `active_class` helper method, the `menu_item` helper method and a bunch more. Also how the filtering works, and more :)

seanmarcia replied:
Also, please pull and re-test. I didn't realize that the /public/assets director was in the gitignore file.

ShamiTomita committed:
plainadmin style change for contact_types

ShamiTomita committed:
plainadmin style change for contact_types

ShamiTomita committed:
removed empty classes

ShamiTomita committed:
add Type and Description + plainadmin error wrap

vasconsaurus committed:
Fixed centering for the images in the main page

The images weren't centered when the width was smaller.
Since this was a small fiz Sean and Shen suggested doing it
this here.

ShamiTomita committed:
added plainadmin title

ShamiTomita committed:
isolated title div from card div

ShamiTomita committed:
index style classes converted to plainadmin

gregblake committed:
Updates mileage_rates index to use plainadmin

Before:
https://user-images.githubusercontent.com/1179668/208957780-92914a55-b2fd-4248-9ef8-09cbe7549d5a.png

After:
https://user-images.githubusercontent.com/1179668/208958002-0790393c-b22d-45f2-b459-1cb13d8060cc.png

Supervisors Index (as a reference):
https://user-images.githubusercontent.com/1179668/208958164-dd83abd4-e06c-423a-8fbb-400347bdc7d3.png

gregblake committed:
Updates mileage_rates form to use plainadmin

Before: (screen shot):

https://user-images.githubusercontent.com/1179668/208975327-19b82116-ba68-40d4-8156-ddb83f3bb757.png

After (screen recording):

https://user-images.githubusercontent.com/1179668/208974932-0d8aeed5-b565-47f4-a619-16f8676775dd.mov

vasconsaurus committed:
wip: added style to buttons, card wrapper, radio buttons

vasconsaurus committed:
wip: fixed the selection, still need to fix placeholder

gregblake committed:
Add new plainadmin theme to the judges form

Screenshots:

* [Before](https://user-images.githubusercontent.com/1179668/208986966-434ec832-e8b6-4a32-96ed-2076e4fcc114.png)
* [After](https://user-images.githubusercontent.com/1179668/208987005-21e66172-a331-4cf4-aec5-a828319440cb.png)

compwron committed:
Merge pull request #4345 from gregblake/apply_new_theme_to_mileage_rates_views

Resolves https://github.com/rubyforgood/casa/issues/4321

compwron committed:
Merge pull request #4346 from gregblake/apply_new_theme_to_judges_views

Resolves https://github.com/rubyforgood/casa/issues/4318

compwron committed:
Merge pull request #4340 from ShamiTomita/4312-new-theme-contact_types-pages

Resolves https://github.com/rubyforgood/casa/issues/4312

gregblake committed:
Add new plainadmin theme to the languages form

Screenshots:

* [Before](https://user-images.githubusercontent.com/1179668/208991777-fb5ccc3b-c0fb-4cff-b183-eb544905dec0.png)
* [After (new)](https://user-images.githubusercontent.com/1179668/208991399-d01fc1ad-ca87-4262-881d-bdbac64d7c16.png)
* [After (edit)](https://user-images.githubusercontent.com/1179668/208991511-4d78a032-61a9-47be-868f-e9e5a70b36c6.png)

seanmarcia committed:
Merge pull request #4347 from gregblake/new_theme_languages_form

Add new plainadmin theme to the languages form

gregblake committed:
Add plainadmin theme: new contact type group form

Screenshots:

* [Before](https://user-images.githubusercontent.com/1179668/208996353-ac9c72c7-c8c3-44a1-a9e5-3cd08bc4ddc1.png)
* [After](https://user-images.githubusercontent.com/1179668/208996405-58fccc96-6c45-4d95-80bf-a8320cf743e5.png)
* [After (form validation)](https://user-images.githubusercontent.com/1179668/208996457-ddb20ed7-ba39-45c1-84a5-e49c4e90377b.png)

seanmarcia committed:
Merge pull request #4348 from gregblake/new_theme_contact_type_group

Add plainadmin theme: new contact type group form

ShamiTomita committed:
casa admin form plainadmin classes and wraps

ShamiTomita committed:
shared(edit&error_mes&invite_log) plainadmin

ShamiTomita committed:
removed table section from casa admin form

ShamiTomita committed:
removed section table wrapper from casaadmin index

vasconsaurus committed:
wip: buttons

vasconsaurus committed:
added prompt: true to show please select

this was how I figure out how to add the please select to
the select fields, however, I had to comment out info that
I'm not sure was ok to comment out

vasconsaurus committed:
removed commented parts

vasconsaurus committed:
updated filter modal

seanmarcia committed:
Merge pull request #4349 from ShamiTomita/4305-new-theme-casa_admins-pages

4305 new theme casa admins pages

seanmarcia committed:
Add hover state to buttons on supervisor index page.

vasconsaurus committed:
updated buttons to include icon and fixed hover

seanmarcia committed:
Update reports.js

seanmarcia committed:
Merge pull request #4350 from rubyforgood/issue4325-reports_page

Applied new theme to reports page

seanmarcia committed:
Make buttons consistent size.

littleforest committed:
Capitalize SMS and CASA

littleforest committed:
Add alt tags to images for accessibility

littleforest committed:
Modify some colors for accessibility

littleforest committed:
Remove ill-placed h2 tag

littleforest committed:
Remove unnecessary closing slash on br tag

littleforest committed:
More color adjustments for accessibility

littleforest committed:
Add h1 and landmark tags

littleforest committed:
Remove gradient since lightest color allowed for
AAA rating is almost identical to darker blue

seanmarcia committed:
Start consolidating js/css

seanmarcia committed:
Start consolidating js/css

seanmarcia committed:
File cleanup;

seanmarcia committed:
Update  devise css

italomatos committed:
refactory index.html.erb to new theme

compwron committed:
Merge pull request #4356 from italomatos/4308-new-theme-case_court_reports-pages

refactory index.html.erb to new theme

seanmarcia committed:
Update footer to use new theme

raychiranjib1 committed:
Applied new theme to user edit page.

BrunoViveiros committed:
Add plainadmin theme to hearing_types pages

Isaac-alencar committed:
add new theme to supevisors view

ShamiTomita committed:
changes

BrunoViveiros committed:
edit submit and new list item buttons

ShamiTomita committed:
lookin good, needs clean up

Learningstuff98 committed:
Added new style for checklist item new and edit pages

seanmarcia committed:
Merge pull request #4365 from BrunoViveiros/new_theme_hearing_types_page

Add plainadmin theme to hearing_types pages

seanmarcia committed:
Fix button size for consistency.

italomatos committed:
fix select2 component and button size

italomatos committed:
fix select2 component and button size

seanmarcia committed:
Merge pull request #4368 from italomatos/4308-fix-case-court-reports-select2-and-generate-button

fix select2 component and button size

ShamiTomita committed:
consistency in mb, removed "and" from hrs/min

italomatos committed:
refactory duties page

compwron committed:
Merge pull request #4377 from italomatos/4323-refactory-other-duties-page

refactory duties page

compwron committed:
Merge pull request #4367 from Isaac-alencar/supervisors-page

add new theme to supevisors view

compwron committed:
Merge pull request #4366 from Learningstuff98/new-theme-for-checklist_item-pages

New style for checklist item new and edit pages

compwron committed:
Merge pull request #4364 from raychiranjib1/feature/f202212224327_new_theme_user_pages

Applied new theme to user edit page.

Learningstuff98 committed:
Adjusted the style for the submit button

ShamiTomita committed:
further plainadmin conversion, removed empty class

italomatos committed:
add icon to submit button

seanmarcia committed:
Merge pull request #4379 from Learningstuff98/style-adjustment-for-checklist_item-submit-button

seanmarcia committed:
Merge pull request #4380 from italomatos/4323-fix-submit-button-icon

bugsmithd committed:
Adds new theme to volunteers pages

bugsmithd committed:
Adds icons to buttons and makes them smaller

Isaac-alencar committed:
fix unassign action style and add icons to buttons

seanmarcia committed:
Merge pull request #4382 from anshu1992/issue-4328/new-theme-volunteers-page

Issue-4328/Adds new theme to volunteers pages

Learningstuff98 committed:
Changed the style for the emancipations index page

bugsmithd committed:
Adds new theme to learning hours pages

seanmarcia committed:
Merge pull request #4385 from anshu1992/issue-4320/add-new-theme-to-learning-hours-pages

Adds new theme to learning hours pages

Learningstuff98 committed:
Made styling changes to the emancipation show page

seanmarcia committed:
Add in a separate sidebar for all_casa_admins as well as remove the header from all_casa_admins

Learningstuff98 committed:
Changed the styling for the fund request form

bugsmithd committed:
Adds new theme to Devise pages

bugsmithd committed:
Adds new theme to Devise pages

seanmarcia committed:
Merge pull request #4386 from anshu1992/issue-4314/add-new-theme-to-devise-pages

bugsmithd committed:
Updates helper methods and sidebar partial to show correct active links

compwron committed:
Merge pull request #4387 from anshu1992/issue-4331/update-helper-methods-for-the-new-theme

Updates helper methods and sidebar partial to show correct active links

seanmarcia committed:
Merge pull request #4393 from Isaac-alencar/supervisors-page

fix unassign action style and add icons to buttons

ShamiTomita committed:
added icons, btn-small class

ShamiTomita committed:
adding icons to previously edited pages

ShamiTomita committed:
removed redundant div

ShamiTomita committed:
lint and additional changes

ShamiTomita committed:
Merge branch 'new_theme' into 4309-new-theme-casa_contacts-pages

seanmarcia committed:
Merge pull request #4381 from ShamiTomita/4309-new-theme-casa_contacts-pages

4309 new theme casa contacts pages

Learningstuff98 committed:
Made some tweaks to the emancipation checklist page

FireLemons committed:
add language description

seanmarcia committed:
Merge pull request #4384 from FireLemons/new_theme

moved language description to under language section

Learningstuff98 committed:
Small tweak

seanmarcia committed:
Merge pull request #4399 from Learningstuff98/new-theme-for-emancipations-and-fund_requests

New theme for emancipation checklists and fund requests

littleforest committed:
Remove some comments and extra line breaks

littleforest committed:
Fix href to be casa instead of human essentials

seanmarcia committed:
Merge pull request #4406 from rubyforgood/4330-new-theme-landing-page

Updates to landing page for new theme

BrunoViveiros committed:
Add plainadmin theme to case_org pages

seanmarcia committed:
Merge pull request #4412 from BrunoViveiros/new_theme-casa_org_pages

Add plainadmin theme to case_org pages

ashstewart7 committed:
Some progress towards updating import pages

bugsmithd committed:
Adds description to contact type groups' form.

tacoda committed:
Update style for note edit view

bugsmithd committed:
Don't show group actions expanded by default

seanmarcia committed:
Merge pull request #4416 from anshu1992/issue-4281/describe-contact-type-groups

Adds description to contact type groups' form.

Isaac-alencar committed:
Add plainadmin theme to casa_cases pages

Isaac-alencar committed:
Add plainadmin theme to casa_cases pages

seanmarcia committed:
Merge pull request #4420 from Isaac-alencar/update_ui_theme_casa_cases

Add plainadmin theme to casa_cases pages

ashstewart7 committed:
Updated imports pages to new theme

ashstewart7 committed:
Tidying up imports pages

BrunoViveiros committed:
Add plainadmin theme to court_dates pages

tacoda committed:
Update style for notifcation index view

ashstewart7 committed:
Remove back to dashboard button

seanmarcia committed:
Merge pull request #4421 from rubyforgood/imports_pages_theme

Convert imports pages to plainadmin theme

tacoda committed:
Update styling on notifications index view

seanmarcia committed:
Merge pull request #4422 from BrunoViveiros/new_theme-court_dates_pages

Add plainadmin theme to court_dates pages

seanmarcia committed:
Remove the beating heart

seanmarcia committed:
Merge pull request #4419 from anshu1992/dont-show-group-actions-expanded-by-default

Don't show group actions expanded by default

seanmarcia committed:
Merge pull request #4424 from tacoda/update-theme-notes-and-notifications

Update theme for notes and notifications

bugsmithd committed:
Show dropdown open when any of the child anchor is active

seanmarcia committed:
Merge pull request #4425 from anshu1992/dont-show-group-actions-expanded-by-default

Show dropdown open when any of the child anchor is active

Learningstuff98 committed:
Changed the theme for the reimbursement queue page

seanmarcia committed:
Merge pull request #4426 from Learningstuff98/new-style-for-reimbursements-pages

Changed the theme for the reimbursement queue page

seanmarcia committed:
Filters were not workings. Fixed them.

seanmarcia committed:
Styling cleanup.

seanmarcia committed:
Lots of formatting and icon fixes.

seanmarcia committed:
Fix buttons and dropdowns on volunteer index page.

seanmarcia committed:
Update casa cases page.

seanmarcia committed:
Update volunteer form and new volunteer.

seanmarcia committed:
Add in csrf token back.

seanmarcia committed:
volunteer index

seanmarcia committed:
Fix filter button on supervisors

seanmarcia committed:
Add icon to generate report

seanmarcia committed:
More icon fix/updates

msespos committed:
Update theme for all casa admins pages v2 (#4436)

* Update theme to use plainadmin theme

* Update theme to use plainadmin theme

* Fix indentation

* Update theme to use plainadmin theme

* Remove duplicate script tag for all_casa_admins to fix DataTables popup bug

* Edit class of 'Active' pill

* Update theme to use plainadmin theme

* Add plainadmin styling for new patch notes and fix JS error from missing menu button in Admin mode

* Add icon to Submit button

* Add icon to Submit button

* Change links to link_to from href and add plus icons

* Add plus icon to button

* Change form.submit to button_tag

* Change form.submit to button_tag

* Change form.submit to button_tag

* Add icon to Create button

rae-stanton committed:
Updated submit button icons (#4445)

Button updates

Fixed the sizing on the judges and language submit buttons and added the checkbox on the buttons.

seanmarcia committed:
Update icons and constants

seanmarcia committed:
More styling fixes.

seanmarcia committed:
Merge branch 'main' into new_theme

seanmarcia committed:
Rebase in master.

seanmarcia committed:
Standardrb and erblint fixes

seanmarcia committed:
Standardrb and erblint fixes

ShamiTomita committed:
Failures 4 through 23 (#4458)

* initial changes

* changes

* alert and notice different, orphan alert div needs home

* header errors at bottom

* cleanup on aisle lint

* removed empty closing div

seanmarcia committed:
Fix brakeman issue.

seanmarcia committed:
Fix brakeman issue.

seanmarcia committed:
Fix brakeman issue.

seanmarcia committed:
Fix brakeman issue.

seanmarcia committed:
Fix brakeman issue.

seanmarcia committed:
Spec fix

seanmarcia committed:
Test fixes

ShamiTomita committed:
Next 15 New Theme Specs (#4494)

* restart

* changed user to volunteer

* session flash

* adding .nil? back, removing it messes with flash messages

* Lint Lint Lint!

seanmarcia committed:
Merge branch 'main' into new_theme

bugsmithd committed:
Fixes specs for new theme (#4535)

seanmarcia committed:
Merge branch 'main' into new_theme

seanmarcia committed:
Fix some copy/paste errors in update

bugsmithd committed:
Fixes next 15 specs for new_theme (#4536)

Learningstuff98 committed:
Fix some tests learningstuff98 (#4516)

* Fixed a test

* Fixed more tests

* Added spec file and a test for the header

* Added more tests

* Added header tests for supervisors

* Added header tests for volunteers

* Reworked notification tests

* Moved impersonation test

* Fixed test for judge form

* Updated a test

* Fixed another test

* updated test

* fixed several tests

---------

Co-authored-by: Sean Marcia <seanmarcia@github.com>

seanmarcia committed:
Lint fix

seanmarcia committed:
Sidebar fix -- should only render if logged in.

seanmarcia committed:
Clean up button and modal.

seanmarcia committed:
Add in the messaging for when impersonating.

seanmarcia committed:
Edit spec fix

seanmarcia committed:
Fix new spec

seanmarcia committed:
More tests fixed

seanmarcia committed:
Fonts for tests

seanmarcia committed:
Revert

seanmarcia committed:
Fonts for tests

seanmarcia committed:
More specs

ShamiTomita committed:
Test-Fixes (#4537)

* minor changes

* export data, ugly fix

seanmarcia committed:
Add select2 to reports page.

seanmarcia committed:
default vaules

ShamiTomita committed:
Final spec fixes for new_theme??? (#4544)

* change casacase back to volunteer

* added missing emancipation-category-check-box class

* changed btn-primary to primary-btn for court_dates

* added type button for additiona expenses

* added match: :first for checklist_items/destroy

* added flash messages for all_casa_admins edit

* removed duplicate flash message on all_casa dashboard

* removed conditional, just added flash messages

* applied anshu's fix for spec sign out

* lint

* addint data-bs to toggle and target for contacts

seanmarcia committed:
Fix tests to use select2

seanmarcia committed:
Yarn uypdate

seanmarcia committed:
lint

ujwalkumar1995 committed:
Delete package-lock file

FireLemons committed:
Merge pull request #4557 from ujwalkumar1995/new_theme

Delete package-lock file

seanmarcia committed:
Fix notification button.

bugsmithd committed:
Fixes edit organization link in admin header (#4579)

seanmarcia committed:
Merge branch 'main' into new_theme

seanmarcia committed:
Yarn

seanmarcia committed:
Popper fix

leslie-seeberger committed:
Ensure warning for disabled js on landing page
issue #4567

dominiclizarraga committed:
edit and button path notes

seanmarcia committed:
Test fix

seanmarcia committed:
Merge branch 'main' into new_theme

seanmarcia committed:
Merge issue

seanmarcia committed:
Title

zacgra committed:
Fix the noscript style bug from #4568 and update styling to make the notification more prominent

FireLemons committed:
Merge pull request #4648 from zacgra/new_theme

Fix the noscript style bug and update notification styling

FireLemons committed:
update branch to include new noscript warning styling

FireLemons committed:
Merge pull request #4604 from leslie-seeberger/4567--javascript-required-warning

Ensure warning for disabled js on landing page

seanmarcia committed:
Merge branch 'main' into new_theme

FireLemons committed:
Merge pull request #4614 from dominiclizarraga/new_theme

edit and delete button patch notes

seanmarcia committed:
Merge branch 'main' into new_theme

seanmarcia committed:
fixed sidebar'

FireLemons commented:
wait why is this closed
